src/backend/services/mining-api/endpoints.md

```diff
@@ -0,0 +1,75 @@
+# Mining
+
+```json
+Transaction Object = {
+    "from" : string,
+    "to" : string,
+    "amount" : string,
+    "id" : string,
+    "signature" : string
+}
+
+```
+
+## Queue a new transaction for mining
+
+> GET http://localhost/mining/queue
+
+Required JSON:
+
+```json
+Transaction Object
+```
+
+Example return:
+
+```json
+200
+
+{
+  "success": True
+}
+```
+
+```json
+400
+
+{
+  "error": string
+}
+```
+
+# Socket events
+
+> client sends "proof"
+
+Required JSON:
+
+```json
+{
+  "id": string, // transaction id for which the proof was found
+  "proof": string,
+  "nonce": string,
+  "minerId": string
+}
+```
+
+> broadcast to all clients broadcasts "findProof"
+
+Sends JSON:
+
+```json
+Transaction Object
```

Comment:
Can you please change this to be the explicit object, I know we use it many other places but always best to be explicit for 3rd party devs

Reply:
I defined the object at the beginning of the file

Reply:
Ah yes I didnt see that before, thanks for pointing that out